plugins/jobtype/src/azkaban/jobtype/HadoopJavaJob.java

```diff
@@ -121,8 +121,27 @@ protected String getJVMArguments() {
     classPath.add(HadoopConfigurationInjector.getPath(getJobProps(),
         getWorkingDirectory()));
 
-    List<String> typeClassPath =
-        getSysProps().getStringList("jobtype.classpath", null, ",");
+    // merging classpaths from plugin.properties
+    mergeClassPaths(classPath,
+      getJobProps().getStringList("jobtype.classpath", null, ","));
+    // merging classpaths from private.properties
+    mergeClassPaths(classPath,
+      getSysProps().getStringList("jobtype.classpath", null, ","));
+
+    List<String> typeGlobalClassPath =
+        getSysProps().getStringList("jobtype.global.classpath", null, ",");
+    if (typeGlobalClassPath != null) {
```

Comment:
Can we call the mergeClassPaths for the logic here ? If it is intended to merge the path as is (w/o the absolute check) we can add a switch to mergeClassPath signature to bypass fit and finish if not intended.